task_ajax/js/classes/albumGetInfo.class.js

```diff
@@ -0,0 +1,12 @@
+function AlbumGetInfo(artist, album, api_key) {
```

Comment:
why artist and album are instance fields? they should be arguments for load function, don't you think so?
you no need to create new instance when you want to load data with another parameters